src/test/scala/it/pagopa/interop/backendforfrontend/AuthorizationApiServiceSpec.scala

```diff
@@ -17,21 +17,28 @@ import com.typesafe.scalalogging.LoggerTakingImplicit
 import it.pagopa.interop.commons.logging.ContextFieldsToLog
 import it.pagopa.interop.commons.ratelimiter.model.RateLimitStatus
 
+import cats.syntax.all._
+import it.pagopa.interop.tenantprocess
 import scala.concurrent.duration.DurationInt
+import it.pagopa.interop.tenantmanagement.client.model.Tenant
+import it.pagopa.interop.commons.utils.service.OffsetDateTimeSupplier
+import it.pagopa.interop.tenantmanagement.client.model.ExternalId
+import it.pagopa.interop.tenantmanagement.client.invoker.ApiError
+import it.pagopa.interop.selfcare.partyprocess.client.model.Institution
 
 class AuthorizationApiServiceSpec extends AnyWordSpecLike with SpecHelper with ScalatestRouteTest {
 
   "Generating a session token" should {
 
-    "succeed" in {
+    "succeed when the tenant is present" in {
 
       val uid: String      = UUID.randomUUID().toString
-      val orgId: UUID      = UUID.randomUUID()
-      val orgIdStr: String = orgId.toString
+      val selfcareId: UUID = UUID.randomUUID()
```

Comment:
why not defining this directly as a String instead of using `toString` everywhere?
Because, you know, this is a `String` 🥲